app/uk/gov/hmrc/fhregistrationfrontend/views/forms/contact_person.scala.html

```diff
@@ -120,7 +120,9 @@
 
 @layout(title = titlePrefix + title, pageScripts = Some(pageScripts), pageHeadBlock = Some(pageHeadBlock)) {
 
-  @if(contactPersonForm.errors.nonEmpty) {
+    <link href="@routes.Assets.at("stylesheets/accessible-autocomplete.min.css")" media="all" rel="stylesheet" type="text/css">
```

Comment:
these should not be positioned within the page body - they should sit in the pageHeadBlock that is passed into layout